### DISCUSSION TITLE
Question about Face Mask

### DISCUSSION THREAD

**Discussion by FinnishNutMilk** [hacksider/Deep-Live-Cam#687] [Q&A]
is it planned to have a feature to be able to change the face mask type to occlusion for example? Allowing objects to go infront of the face or in the mouth without the mask coming off? The project is very good!

**Comment by hacksider** [comment]
It is planned but the priority is the webcam options. After that is the optimization. But it will be on the top 10 features. It all depends on the needs of the community or the help of the community as well. As of now I'm funding the webcam options personally on the bounty. ☺️

**Reply by FinnishNutMilk** [reply]
okay! :)

**Comment by hacksider** [comment]
Implemented on the current build

**Reply by hacksider** [reply]
https://github.com/hacksider/Deep-Live-Cam/pull/740
